src/Controls/src/Core/HandlerImpl/WebView/WebView.Windows.cs

```diff
@@ -1,9 +1,21 @@
-﻿using Microsoft.Maui.Controls.Platform;
-using Microsoft.Maui.Controls.PlatformConfiguration.WindowsSpecific;
+﻿#nullable enable
+using System;
 
 namespace Microsoft.Maui.Controls
 {
 	public partial class WebView
 	{
+		protected override void OnPropertyChanging(string? propertyName = null)
+		{
+			base.OnPropertyChanging(propertyName);
+
+			if (propertyName == nameof(Window) && Window is not null)
```

Comment:
Minor: a null check is cheaper than a string comparison, switch the && around will allow you to get out of the if earlier.
```suggestion
			if (Window is not null && propertyName == nameof(Window))
```